snap/snapcraft.yaml

```diff
@@ -14,6 +14,13 @@ environment:
   JAVA_HOME: $SNAP/usr/lib/jvm/java-17-openjdk-amd64
   PATH: $JAVA_HOME/bin:$PATH
 
+slots:
+  logs:
+    interface: content
+    source:
+      read:
+        - $SNAP_COMMON/log
```

Comment:
**question:** Did we decide on `log` or `logs`? Looking at the spec it seems like the latter.

Reply:
Ah but right now it's `log`. OK. Will approve!

Reply:
You could also update `local/start-kafka-wrapper.bash` to change `LOG_DIR="${SNAP_COMMON}/logs", but maybe worth doing later.

Reply:
I'd probably do all refactoring in a single PR about implementing the structure we defined in the spec. 

there were already things I could have done, but I believe it is best to keep them confined in the ticket. This PR is really only for implementing the plug/slot content interface